admin/config/environments/production/server.json

```diff
@@ -0,0 +1,14 @@
+{
+  "host": "localhost",
+  "port": "${process.env.PORT || 1337}",
```

Comment:
i feel to have this work on production we are going to have to update this config file 🤔 

@jsegal205